src/main/kotlin/de/klg71/keycloakmigration/changeControl/actions/group/RenameGroupAction.kt

```diff
@@ -0,0 +1,43 @@
+package de.klg71.keycloakmigration.changeControl.actions.group
+
+import de.klg71.keycloakmigration.changeControl.actions.Action
+import de.klg71.keycloakmigration.changeControl.actions.MigrationException
+import de.klg71.keycloakmigration.keycloakapi.existsGroup
+import de.klg71.keycloakmigration.keycloakapi.groupByName
+import de.klg71.keycloakmigration.keycloakapi.model.Group
+import de.klg71.keycloakmigration.keycloakapi.model.UpdateGroup
+
+class RenameGroupAction(
+    realm: String? = null,
+    private val name: String,
+    private val newName: String) : Action(realm) {
+
+    private lateinit var group: Group
+
+    private fun updateGroup() = UpdateGroup(newName, group.path, group.attributes,
+        //FIXME
```

Comment:
Could you elaborate how a fix could be implemented here?

Reply:
this is pretty much copy paste from https://github.com/mayope/keycloakmigration/blob/master/src/main/kotlin/de/klg71/keycloakmigration/changeControl/actions/group/UpdateGroupAction.kt 

that's where suggestion to expand `updateGroup` comes from :)

Reply:
Ok than it is obviously on me to fix this ^^